CREDITS

```diff
@@ -1 +1,2 @@
 The phpAmazonMWS library was designed and written by Thomas Hernandez (peardian at gmail) for the CPI Group.
+The v1.4.0 refactoring when adding the AmazonMWSConfig class was done by Steve Childs (stevechilds76 at gmail) for Color Confidence (UK).
```

Comment:
I do want to give contributors credit, but I haven't yet figured out the best way to go about it, so for now I would prefer it if the credits file was not modified. You are welcome to put an author tag in the new config class, though.